src/reason-parser/reason_lexer.mll

```diff
@@ -538,6 +526,11 @@ rule token = parse
         let esc = String.sub l 1 (String.length l - 1) in
         raise (Error(Illegal_escape esc, Location.curr lexbuf))
       }
+  | "#=<" {
```

Comment:
Could this never otherwise be a valid infix/ppx extension token on its own? Do we want to require that what follows the `<` is an identifier without any white space in between?